addons/project_todo/views/project_task_views.xml

```diff
@@ -54,17 +52,16 @@
                                         <field name="displayed_image_id" widget="attachment_image"/>
                                     </div>
                                 </div>
-                                <div class="o_kanban_record_bottom">
-                                    <div class="oe_kanban_bottom_left"/>
-                                    <div class="oe_kanban_bottom_right"  t-att-class="['1_done', '1_canceled'].includes(record.state.raw_value) ? 'opacity-50' : ''">
-                                        <div t-attf-class="d-flex #{todoHasAssignees ? 'w-100 align-items-center justify-content-end' : 'align-items-end'}">
-                                            <div class="d-flex align-items-center me-2 mt-2">
-                                                <div t-if="todoHasAssignees">
-                                                    <field name="user_ids" widget="many2many_avatar_user"/>
-                                                </div>
-                                                <field name="activity_ids" widget="kanban_activity"/>
-                                            </div>
+                                <div class="o_kanban_record_bottom mt-2">
+                                    <div class="oe_kanban_bottom_left" t-att-class="['1_done', '1_canceled'].includes(record.state.raw_value) ? 'opacity-50' : ''">
+                                        <field name="priority" widget="priority"/>
+                                        <field name="activity_ids" widget="kanban_activity" style="padding-top: 1.5px;"/>
+                                    </div>
+                                    <div class="oe_kanban_bottom_right">
+                                        <div t-att-class="{'opacity-50': ['1_done', '1_canceled'].includes(record.state.raw_value), 'o_todo_hide_avatar': !todoHasAssignees}">
+                                            <field name="user_ids" widget="many2many_avatar_user" domain="[('share', '=', False)]"/>
```

Comment:
the domain is now defined in the field definition and so it is no longer needed. 🙂